utils/bigquery.js

```diff
@@ -1,18 +1,33 @@
 const {BigQuery} = require('@google-cloud/bigquery');
 const bigquery = new BigQuery();
 
-const getTable = async (tableName, isParent, siteCode) => {
+
+const getTable = async (tableName, isParent, siteCode, status, recruit) => {
```

Comment:
**status** and **recruit** are not used, so we can remove them.

Reply:
oops i removed them in my functions folder forgot to remove them on the utils folder..